src/views/tree-view.ts

```diff
@@ -16,9 +16,7 @@ export class TreeDataProvider implements vscode.TreeDataProvider<TreeItem> {
         new TreeItem("Stop", undefined, "pros.stop"),
         new TreeItem("Brain Terminal", undefined, "pros.terminal"),
         new TreeItem("Integrated Terminal", undefined, "pros.showterminal"),
-        new TreeItem("Capture Image", undefined, "pros.capture"),
-        new TreeItem("Set Team Number", undefined, "pros.teamnumber"),
-        new TreeItem("Set Robot Name", undefined, "pros.robotname"),
```

Comment:
not sure about these getting removed either, more tech support from kids asking where it went lol